pwndbg/commands/heap.py

```diff
@@ -119,19 +130,33 @@ def arenas():
     main_arena_addr = int(arena.address)
     fmt = '[%%%ds]' % (pwndbg.arch.ptrsize *2)
     while addr != main_arena_addr:
-        
+
         h = heap.get_region(addr)
         if not h:
             print(message.error('Could not find the heap'))
             return
 
         hdr = message.hint(fmt % (hex(addr) if addr else 'main'))
         print(hdr, M.heap(str(h)))
-        addr = int(arena['next'])        
+        addr = int(arena['next'])
         arena = heap.get_arena(addr)
 
 
-    
+@pwndbg.commands.ParsedCommand
+@pwndbg.commands.OnlyWhenRunning
+def tcache():
+    """
+    Prints out the thread cache.
+    """
+    main_heap   = pwndbg.heap.current
+    tcache  = main_heap.thread_cache
+
+    if tcache is None:
+        return
```

Comment:
Would be good to display some info/warning/whatever

Reply:
ptmalloc.Heap->thread_cache displays one already.